.github/workflows/include-kcl-checksums.yml

```diff
@@ -0,0 +1,31 @@
+name: Include KCL Modules Checksum
+
+on:
+  push:
```

Comment:
This github action should not be triggered automatically by the push, it should be set to be triggered manually, and it is best to provide input to set `export KPM_REG="localhost:5001"
export KPM_REPO="test"` 

This gives us more control over how the action runs, rather than having to refresh the checksum every time we push, which would introduce unnecessary risks

Reply:
Sure, I can modify the workflow to trigger manually. However, I didn’t understand the purpose of introducing `KPM_REG` and `KPM_REPO`, since we only need to log in to `ghcr.io`, and within the `integrate-checksum/main.go` file, we are directly referring to `ghcr.io` as the registry and `kcl-lang` as the repository.

Reply:
Never mind, I understood from the KPM code that the settings load environment variables.

Reply:
PTAL, I also added a new secret, `REGISTRY_ACCESS_TOKEN`, so that we can specify the registry password depending on the registry we are using.